src/cmd/manage_nsfs.js

```diff
@@ -17,51 +17,27 @@ const ManageCLIResponse = require('../manage_nsfs/manage_nsfs_cli_responses').Ma
 const manage_nsfs_glacier = require('../manage_nsfs/manage_nsfs_glacier');
 const nsfs_schema_utils = require('../manage_nsfs/nsfs_schema_utils');
 const { print_usage } = require('../manage_nsfs/manage_nsfs_help_utils');
-const { TYPES, ACTIONS, LIST_ACCOUNT_FILTERS, LIST_BUCKET_FILTERS,
+const { TYPES, ACTIONS, CONFIG_SUBDIRS,
+    LIST_ACCOUNT_FILTERS, LIST_BUCKET_FILTERS,
     GLACIER_ACTIONS } = require('../manage_nsfs/manage_nsfs_constants');
 const { throw_cli_error, write_stdout_response, get_config_file_path, get_symlink_config_file_path,
-    get_config_data, get_boolean_or_string_value, has_access_keys, set_debug_level, get_config_data_if_exists } = require('../manage_nsfs/manage_nsfs_cli_utils');
+    get_config_data, get_boolean_or_string_value, has_access_keys, set_debug_level, get_config_data_if_exists,
+    check_and_create_config_dirs } = require('../manage_nsfs/manage_nsfs_cli_utils');
 const manage_nsfs_validations = require('../manage_nsfs/manage_nsfs_validations');
 const nc_mkm = require('../manage_nsfs/nc_master_key_manager').get_instance();
 
-const buckets_dir_name = '/buckets';
-const accounts_dir_name = '/accounts';
-const access_keys_dir_name = '/access_keys';
-const acounts_dir_relative_path = '../accounts/';
-
-let config_root;
-let accounts_dir_path;
-let access_keys_dir_path;
-let buckets_dir_path;
-let config_root_backend;
-
-async function check_and_create_config_dirs() {
-    const pre_req_dirs = [
-        config_root,
-        buckets_dir_path,
-        accounts_dir_path,
-        access_keys_dir_path,
-    ];
-
-    if (config.NSFS_GLACIER_LOGS_ENABLED) {
-        pre_req_dirs.push(config.NSFS_GLACIER_LOGS_DIR);
-    }
-
-    for (const dir_path of pre_req_dirs) {
-        try {
-            const fs_context = native_fs_utils.get_process_fs_context(config_root_backend);
-            const dir_exists = await native_fs_utils.is_path_exists(fs_context, dir_path);
-            if (dir_exists) {
-                dbg.log1('nsfs.check_and_create_config_dirs: config dir exists:', dir_path);
-            } else {
-                await native_fs_utils._create_path(dir_path, fs_context, config.BASE_MODE_CONFIG_DIR);
-                dbg.log1('nsfs.check_and_create_config_dirs: config dir was created:', dir_path);
-            }
-        } catch (err) {
-            dbg.log1('nsfs.check_and_create_config_dirs: could not create pre requisite path', dir_path);
-        }
-    }
-}
+const nsfs_config = Object.seal({
```

Comment:
let's rename `nsfs_config` to `global_config` since we want to remove `nsfs` references